src/types/constants.ts

```diff
@@ -50,6 +50,27 @@ const constants = {
   ],
   org_roles: ['org_admin', 'study_designer', 'study_coordinator'] as const,
 
+  appBranding: [
+    {
+      appId: OPEN_BRIDGE_APP_ID,
+      title: 'Open Bridge',
+      logo: 'open_bridge',
+      iconPrefix: '',
+    },
+    {
+      appId: ARC_APP_ID,
+      title: 'ARC',
+      logo: 'arc',
+      iconPrefix: 'arc_',
+    },
+    {
+      appId: INV_ARC_ID,
```

Comment:
Should this be INV_ARC_APP_ID for consistency?

Reply:
Probably. Renaming is kinda kludgy in React though so I'd prefer not to blanket rename it. I can add a TODO.